### PR TITLE
Add HNPWA entry using Polymer Starter Kit

### PR DESCRIPTION
**Pull request by mozeal**
Submit for hacker-news-pwas

### CODE REVIEW

File: site/_apps/hn-pwa.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+other-details: ''
+authors: 
+    - name: 'Mozeal'
+lighthouse: 91/100
+interactive-em: 5.88s
+interactive-faster-3g: 4.24s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170604_0Q_306ecb0bc3f53815f01357f8fa6e1cbb&run=2
+wpt-em-link: https://www.webpagetest.org/result/170604_3H_17cf72d4b241ea2d5fae6142e860ef64/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170604_0Q_306ecb0bc3f53815f01357f8fa6e1cbb/
+image: /asset/images/hn-pwa-mobile.png
```
Comment (housseindjirdeh) [inline site/_apps/hn-pwa.txt]: Little type here, needs to reference the `assets`not `asset`.

File: site/_apps/hn-pwa.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+scaffolding: 'Polymer CLI & Polymer Starter Kit'
+module-bundling: 'Polymer-build with HTML Imports'
+service-worker: 'Application Shell + data caching with sw-precache'
+perfomance-patterns: 'PRPL, code-splitting for granular loading'
+server-side-rendering: 'None'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Google AppEngine'
+other-details: ''
+authors: 
+    - name: 'Mozeal'
```
Comment (housseindjirdeh) [inline site/_apps/hn-pwa.txt]: So in order for the contributor to show up underneath their app, you'll need to make sure the name here matches that in the contributor submission (for example if you wanted to use the name `Mozeal`, you'll have to put that exact string here _and_ in your submission under `_contributors`

**Comment by addyosmani** [comment]
Friendly ping with respect to the review comments @mozeal :)

**Comment by addyosmani** [comment]
@mozeal do you think you might have time to take a look at the above feedback comments?

**Comment by addyosmani** [comment]
friendly ping :)

**Comment by mozeal** [comment]
I’m so sorry for lately reply.
just re-submitted.

Best
Moz

**Comment by addyosmani** [comment]
@mozeal One last thing: could you double check the timings in your submission are using the numbers for "first interactive" from the Lighthouse reports in WebPageTest?

e.g Go to your report:

image

Click the Lighthouse score:

image


Use the numbers from the performance section:

image

Right now for example the faster 3G and EM numbers do not appear to match those in these reports.

**Comment by addyosmani** [comment]
In case he has time, I'm also looping in @anubhav7495 for his thoughts on this implem :)

**Comment by anubhav7495** [comment]
@mozeal The app looks a little different but that is mostly fine for now.

1. I think the app is missing pagination, which is kind of an important requirement. **[High]**
2. Opening comments view results in a stray `console.log`. **[Medium]**
3. The url scheme is a little odd as `/list/TopStories` or `/list/Comment`, you can simply change it to `/TopStories` or `/Comment`. **[low]**

Please try to fix `High` or `Medium` marked issues for submission. `low` marked issues can be done later.

#### Lighthouse Audits
[Faster 3G](https://www.webpagetest.org/lighthouse.php?test=170723_X2_7c3cabad6a02568429102e66e1cafbe8&run=2#best-practices) `TTI 4s`
[3G EM](https://www.webpagetest.org/lighthouse.php?test=170723_Q9_89ce695791985cb51d4b18f3d538ef21&run=2) `TTI 4.75s`

The above lighthouse audits also highlight some opportunities to improve the performance even further, please try to take a look at that too.

Thanks.

**Comment by addyosmani** [comment]
Friendly ping @mozeal  in case you have time to work on those last remaining bits. We'd love to have this implementation land.

**Comment by addyosmani** [comment]
Hey folks. As it's been a few quarters since the last updates on this PR, we're going to go ahead and close it. If we would like to explore this PR once again, please update the weights based on master and tackle the comments from @agubler in https://github.com/tastejs/hacker-news-pwas/pull/55#issuecomment-317269596. Thanks once again for your efforts to contribute here!

**Pull request closed without merging**